simtools/model/site_model.py

```diff
@@ -117,7 +117,10 @@ def get_array_elements_for_layout(self, layout_name):
             if layout["name"] == layout_name.lower():
                 return layout["elements"]
         self._logger.error(
-            "Array layout '%s' not found in '%s' site model.", layout_name, self.site
+            "Array layout '%s' not found in '%s' site model. Possible choices are '%s'",
+            layout_name,
```

Comment:
The number of layouts will be very large in future; which will make this error message very hard to read. There is a dedicated application to get those kind of information from the database, I think we should keep this away from error message.

Reply:
I included this because I needed to quickly consult it, so I thought it would be good to pass on the information to the logger as well. But I can remove it now.